editor/plugins/font_config_plugin.cpp

```diff
@@ -922,7 +922,8 @@ void FontPreview::_notification(int p_what) {
 						name = vformat("%s (%s)", prev_font->get_font_name(), prev_font->get_font_style_name());
 					}
 					if (prev_font->is_class("FontVariation")) {
-						name += " " + TTR(" - Variation");
+						// TRANSLATORS: This refers to variable font config, appended to the font name.
```

Comment:
Unsure if saying "variable" makes this more or less confusing given the programming context of a "variable".